src/components/overviewPage/overviewPage.stories.tsx

```diff
@@ -26,33 +26,6 @@ export default {
   },
 };
 
```

Comment:
default export is missing `ComponentMeta` typing and other stories are missing `ComponentStory`